concord-server/server/src/main/yml/server.yml

```diff
@@ -1,5 +1,7 @@
 jwtTokenSecret: jkfhgkjfdhgur8943flkds3@#$^UYGJHKJHKK
 
+consensusLevel: 1

```

Comment:
Currently at 1 so we can test the logic with just two users - we want it set to 3 for production.